wgpu-core/src/command/clear.rs

```diff
@@ -228,6 +220,23 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
             });
         }
 
+        let clear_usage = if dst_texture
+            .hal_usage
+            .contains(hal::TextureUses::DEPTH_STENCIL_WRITE)
+        {
+            hal::TextureUses::DEPTH_STENCIL_WRITE
+        } else if dst_texture
+            .hal_usage
+            .contains(hal::TextureUses::COLOR_TARGET)
+        {
+            hal::TextureUses::COLOR_TARGET
+        } else if dst_texture.hal_usage.contains(hal::TextureUses::COPY_DST) {
+            hal::TextureUses::COPY_DST
+        } else {
+            panic!("Every texture must at least have one of hal::TextureUses::DEPTH_STENCIL_WRITE/COLOR_TARGET/COPY_DST. 
```

Comment:
The message here is not making it clear whose fault this is. I believe it's the fault of wgpu-core, since it controls how textures are created. So it's more like `unreachable!` here than a panic, although the panic is better if zero-initialization becomes optional. Anyway, if we refactor this into `expect()` like I suggested we can have a slightly better message.